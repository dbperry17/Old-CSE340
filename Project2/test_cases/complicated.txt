Snape	9
Weasley	6
Weasley	1
Lupin	10
Weasley 1
Sprout	15
Malfoy	5
Dobby	28
Longbottom	30
Potter	31
Weasley	11
Weasley	22
Granger	19
McGonagall	4
Flitwick	17
Weasley	30
Weasley	29
Hagrid	6
Weasley	12
