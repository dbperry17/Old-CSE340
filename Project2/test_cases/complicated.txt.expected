Dobby 28
Flitwick 17
Granger 19
Hagrid 6
Longbottom 30
Lupin 10
Malfoy 5
McGonagall 4
Potter 31
Snape 9
Sprout 15
Weasley 1
Weasley 1
Weasley 6
Weasley 11
Weasley 12
Weasley 22
Weasley 29
Weasley 30
